### PR TITLE
Added say Command

### PR DESCRIPTION
**Pull request by humboldt123**

### CODE REVIEW

**Comment by humboldt123** [comment]
Ignore the first four I messed up my git thing 🙄

**Comment by 5HT2** [comment]
Reset to the fabric branch, you're committing changes to blink in this pull

File: src/main/kotlin/me/zeroeightsix/kami/feature/command/SayCommand.kt [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import com.mojang.brigadier.CommandDispatcher
+import me.zeroeightsix.kami.mc
+import net.minecraft.server.command.CommandSource
+
+object SayCommand : Command() {
+    override fun register(dispatcher: CommandDispatcher<CommandSource>) {
+        dispatcher register rootLiteral("say") {
+            greedyString("message") {
+                does {
+                    mc.player?.sendChatMessage(it.input.removeRange(0, 4))
```
Comment (5HT2) [inline src/main/kotlin/me/zeroeightsix/kami/feature/command/SayCommand.kt]: what is 0 and 4 for?

Comment (humboldt123) [reply src/main/kotlin/me/zeroeightsix/kami/feature/command/SayCommand.kt]: Image the message `.say I like fruits!`. `it.input.` would get us `.say I like fruits!` but `it.input.removeRange(0, 4)` slices off the first four characters and will yield only `I like fruits!`

Comment (5HT2) [reply src/main/kotlin/me/zeroeightsix/kami/feature/command/SayCommand.kt]: Shouldn't you get the context of the greedy string and send that instead?

Comment (humboldt123) [reply src/main/kotlin/me/zeroeightsix/kami/feature/command/SayCommand.kt]: This is less lines of code and the first four characters will always, *always* be `.say `. Its like relying on the sum of 2 sides of a triangle being greater than the third.

Comment (zeroeightysix) [reply src/main/kotlin/me/zeroeightsix/kami/feature/command/SayCommand.kt]: And it's also terrible code. 
As pointed out by dominika, it's dubious: 
* seemingly random magic values (sure, they have meaning, but it's not pretty)
* it's error-prone (what if we change the command name? what if we introduce an alias? what if we add a terminal, where the prefix is not necessary?)

Just use `"message" from it`.

**Review by zeroeightysix — CHANGES_REQUESTED** [review]

**Review by zeroeightysix — APPROVED** [review]

**Comment by zeroeightysix** [comment]
Thanks!

**Pull request merged**